fe2-android/app/src/main/java/com/github/dedis/student20_pop/utility/handler/ElectionHandler.java

```diff
@@ -0,0 +1,180 @@
+package com.github.dedis.student20_pop.utility.handler;
+
+import static com.github.dedis.student20_pop.model.event.EventState.CLOSED;
+import static com.github.dedis.student20_pop.model.event.EventState.OPENED;
+import static com.github.dedis.student20_pop.model.event.EventState.RESULTS_READY;
+
+import android.util.Log;
+import com.github.dedis.student20_pop.model.Election;
+import com.github.dedis.student20_pop.model.Lao;
+import com.github.dedis.student20_pop.model.WitnessMessage;
+import com.github.dedis.student20_pop.model.data.LAORepository;
+import com.github.dedis.student20_pop.model.network.method.message.data.Action;
+import com.github.dedis.student20_pop.model.network.method.message.data.Data;
+import com.github.dedis.student20_pop.model.network.method.message.data.ElectionResultQuestion;
+import com.github.dedis.student20_pop.model.network.method.message.data.election.CastVote;
+import com.github.dedis.student20_pop.model.network.method.message.data.election.ElectionResult;
+import com.github.dedis.student20_pop.model.network.method.message.data.election.ElectionSetup;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+/** Election messages handler class */
+public class ElectionHandler {
+
+  public static final String TAG = ElectionHandler.class.getSimpleName();
+
+  public static final String ELECTION_NAME = "Name : ";
+  public static final String MESSAGE_ID = "Message ID : ";
+  public static final String ELECTION_ID = "Election ID : ";
+  public static final String ELECTION_QUESTION = "Question : ";
+  public static final String ELECTION_SETUP = "New Election Setup ";
```

Comment:
It might make sense to move these constants (and the `WitnessMessage` creation) in a dedicated function.
Same applies to other handlers.